crates/router/src/core/recon.rs

```diff
@@ -39,7 +39,8 @@ pub async fn send_recon_request(
         .attach_printable("Failed to convert recipient's email to UserEmail")?,
         settings: state.conf.clone(),
         subject: format!(
-            "Dashboard Pro Feature Request by {}",
+            "{} {}",
+            consts::user::EMAIL_SUBJECT_DASHBOARD_REQUEST,
```

Comment:
Can we move the constants related to Recon files in to this file instead
https://github.com/juspay/hyperswitch/blob/5912936f9f2178f002c96e25636b25f532b1ecb2/crates/router/src/consts.rs#L137.